packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -26,18 +26,29 @@ export const RequestLineEdit = ({
   mode,
   item,
 }: RequestLineEditProps) => {
-  const isDisabled = useIsRequestRequisitionDisabled();
+  const disabled = useIsRequestRequisitionDisabled();
   const { Modal } = useDialog({ onClose, isOpen });
   const [currentItem, setCurrentItem] = useBufferState(item);
   const { draft, isLoading, save, update } =
     useDraftRequisitionLine(currentItem);
+  const { next, isDisabled } = useNextRequestLine(currentItem);
```

Comment:
same comment as for response reqs: `{ next, hasNext }` and reverting the name of `disabled => isDisabled` would be clearer overall